finos/git-proxy
# feat: allow ports to be customised via env vars

Closes #378 

To test this PR:

```
npm install -g
export GIT_PROXY_UI_PORT="5000"
export GIT_PROXY_SERVER_PORT="9090"
git-proxy
```

You should see:
```
Service Listening on 5000
Listening on 9090
```
